mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -99,6 +99,91 @@ msgstr "Rhincus Fossil"
 msgid "rhincus_fossil_description"
 msgstr ""
 
+msgid "nu_phone"
+msgstr "Nu Phone"
+
+msgid "nu_phone_description"
+msgstr "You can do a lot with this, e.g. phone home."
+
+msgid "lethe_pill"
+msgstr "Lethe Pill"
+
+msgid "lethe_pill_description"
+msgstr "It resembles an eye and it makes forget about a random technique."
```

Comment:
Bad grammar

"It resembles an eye and it makes a monster forget about a random technique."

Reply:
right, done